ReaderClass/Visiter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ReaderClass
+{
+    public class Visiter: Reader
```

Comment:
Лучше использовать англ. имена без ошибок
```suggestion
    public class Visitor: Reader
```